package.json

```diff
@@ -106,8 +106,8 @@
     "semantic-release": "^19.0.3",
     "simple-git": "^3.15.1",
     "source-map-support": "^0.5.21",
-    "sqlite3": "^5.1.4",
-    "typescript": "~4.9.4"
+    "sqlite3": "^5.1.6",
```

Comment:
Skip updating sqlite3 for now. Due to this issue: https://github.com/TryGhost/node-sqlite3/issues/1694

Reply:
Otherwise this will be giving a high vulnerability issue